doc/2/api/controllers/document/search/index.md

```diff
@@ -33,6 +33,12 @@ Koncorde filters will be translated into an Elasticsearch query.
 Koncorde `bool` operator and `regexp` clause are not supported for search queries.
 :::
 
+<SinceBadge version="auto-version">
```

Comment:
Can you add a section here so we can link a specific anchor to the multi search feature?